distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -0,0 +1,39 @@
+= ${branding-expanded} Documentation
+include::${project.build.directory}/doc-contents/_contents/config.adoc[]
+
+= Introduction
+
+== About ${branding}
+include::{adoc-include}/_introduction/introduction-contents.adoc[]
+
+include::{adoc-include}/_introduction/applications-contents.adoc[]
+
+== Documentation Guide
+
+include::{adoc-include}/_introduction/documentation-notes-contents.adoc[]
+
+<<<
+
+= Core Concepts
```

Comment:
I think we might want to add a section before Metadata about Search.  I am wondering if we need an ingest and content section as well.  

We also need to clearly explain how the system transforms requests from one standard/endpoint to another standard/source.